src/App.js

```diff
@@ -2,38 +2,57 @@ import React, { Component } from 'react';
 import movieData from './sample-data';
 import MovieContainer from './components/Movie-Container/Movie-Container'
 import SelectedMovie from './components/Selected-Movie/Selected-Movie';
-
+import Error from './components/Error/Error'
+import './App.css'
 class App extends Component {
   constructor() {
     super();
     this.state = {
       movies: [],
       movieSummary: false,
-      selectedMovie: ''
+      selectedMovie: '',
```

Comment:
Is `selectedMovie` always going to be a string? If so, then this is fine, but if it changes to an object, then it's better to be consistent with the types and start out as an empty object.